cloud/migrate-mst-cloud/schema-then-data.md

```diff
@@ -0,0 +1,336 @@
+# Migrate schema and data separately
+Migrate larger databases by migrating your schema first, then migrating the
+data. This method copies each table or chunk separately, which allows you to
+restart midway if one copy operation fails.
+
+The procedure to migrate your database requires these steps:
+*   [Migrate schema pre-data](#migrate-schema-pre-data)
+*   [Restore hypertables in Timescale Cloud](#restore-hypertables-in-timescale-cloud)
+*   [Copy data from Managed Service for TimescaleDB](#copy-data-from-managed-service-for-timescaledb)
+*   [Restore data into Timescale Cloud](#restore-data-into-timescale-cloud)
+*   [Migrate schema post-data](#migrate-schema-post-data)
+*   [Recreate continuous aggregates](#recreate-continuous-aggregates) (optional)
+*   [Recreate policies](#recreate-policies) (optional)
+*   [Recompress previously compressed data](#recompress-data) (optional)
+
+<highlight type="warning">
+Depending on your database size and network speed, steps that involve copying
+data can take a very long time. You can continue reading from your old database
+during this time, though performance could be slower. If you writes to your
+old tables during the migration, the new writes might not be transferred to
+Timescale Cloud. To avoid this problem, see
+[migrating an active database](http://docs.timescale.com/cloud/latest/migrate-mst-cloud/#migrating-an-active-database).
+</highlight>
+
+## Prerequisites
+Before you begin, check that you have:
+*   Installed the PostgreSQL [`pg_dump`][pg_dump] and [`pg_restore`][pg_restore]
+    utilities.
+*   Installed a client for connecting to PostgreSQL. These instructions use
+    [psql][psql], but any client works.
+*   Created a new empty database in Timescale Cloud. For more information, see the    
+    [Install Timescale Cloud section][install-timescale-cloud].
+*   Checked that you're running the same major version of PostgreSQL on both
+    Managed Service for TimescaleDB, and Timescale Cloud. For more information about upgrading PostgreSQL on Managed Service for TimescaleDB, see the
+    [upgrading PostgreSQL section][upgrading-postgresql].
+*   Checked that you're running the same major version of TimescaleDB on both
+    Managed Service for TimescaleDB and Timescale Cloud. For more information,
+    see the [upgrading TimescaleDB section][upgrading-timescaledb].
+
+<highlight type="note">
+To speed up migration, compress your data. You can compress any chunks where
+data is not being currently inserted, updated, or deleted. When you finish the
+migration, you can decompress chunks as needed for normal operation. For more
+information about compression and decompression, see the
+[compression section](https://docs.timescale.com/timescaledb/latest/how-to-guides/compression/).
+</highlight>
```

Comment:
@lukas-bernert Does compressing your data prior to migration add any value for the schema-then-data method? I assumed not because `\COPY` decompresses the data anyway, but I don't actually know

Reply:
correct, using \COPY decompresses data